x-pack/plugins/cases/server/plugin.ts

```diff
@@ -150,19 +130,18 @@ export class CasePlugin {
       actionsPluginStart: plugins.actions,
     });
 
-    const getCasesClientWithRequestAndContext = async (
-      context: CasesRequestHandlerContext,
-      request: KibanaRequest
-    ): Promise<CasesClient> => {
+    const client = core.elasticsearch.client;
```

Comment:
We can avoid requiring the context in the function call be retrieving the elasticsearch client from `CoreStart`